content/en/docs/_index.md

```diff
@@ -8,13 +8,13 @@ Flux is a tool for keeping Kubernetes clusters in sync with sources of
 configuration (like Git repositories), and automating updates to
 configuration when there is new code to deploy.
 
-Flux version 2 ("Flux v2") is built from the ground up to use Kubernetes'
+The new version of Flux is built from the ground up to use Kubernetes'
```

Comment:
```suggestion
Flux is built from the ground up to use Kubernetes'
```

We shouldn't have to clarify new or old, in this context we are covering the current version of Flux.